src/CommunityVoices/App/Api/View/DisplayError.php

```diff
@@ -23,14 +25,78 @@ protected function getError($request)
 
         return $response;
     }
-    protected function getAllErrors($request)
+
+    protected function getAllErrors($request,$lines=500)
     {
-        $errors = explode('[]',str_replace('\n','',file_get_contents('/var/www/html/log/access.log')));
-        foreach($errors as &$item) {
-            $item = array("item" => $item);
-        }
-        $response = new HttpFoundation\JsonResponse(["errors" => $errors]);
-        return $response;
+        $dates = empty($request->query->get('dateRange')) ? [false,false] : explode(" - ",$request->query->get('dateRange'));
+        $linePos = empty($request->query->get('linePos')) ? PHP_INT_MAX : intval($request->query->get('linePos'));
+        $numLinesToAdd = intval($request->query->get('numLines')) ?? 0;
+        $initialRequest = $request->query->get('firstTime');
```

Comment:
What is this used for?  Seems a little weird to have a query string activate for a first time, when usually lack thereof would mean first time.